geo.py

```diff
@@ -480,8 +490,153 @@ def parse_address_string(addrstr):
     return addr
 
 
-if __name__ == "__main__":
+def iceprep_for_street(street_name):
+    """ Return the right preposition ("í" or "á") for
+        an Icelandic street name, e.g. "Fiskislóð" """
+
+    # TODO: Implement me properly
+    isuffixes = frozenset(("brekka", "ás", "holt", "tún"))
```

Comment:
Should probably be a global constant tuple: `_I_SUFFIXES` or something similar, no need for `frozenset`